mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -538,6 +545,80 @@ impl Nexus {
         }
     }
 
+    fn nanosecond_timestamp(&self) -> u64 {
+        let timestamp_now = SystemTime::now()
+            .duration_since(SystemTime::UNIX_EPOCH)
```

Comment:
I'm not sure. It might be useful to have errors recorded as an absolute time so it could be compared with events logged on the child - likely to be a different machine - though the timestamps aren't used for that purpose now. Happy to go either way with that.

Reply:
Comparing timestamps with a different machine is even trickier because the clocks won't be synchronised. Linux blktrace/blkparse uses relative timestamps though that's strictly single node. @gila what would be requirements be in this case?

Reply:
Ok, I'm convinced, especially as the main purpose is errors over an interval back from the present. Will change.